x/auctions/client/cli/query.go

```diff
@@ -0,0 +1,84 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/evmos/evmos/v19/x/auctions/types"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd returns the cli query commands for the auctions module.
+func GetQueryCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the auctions module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(
+		GetAuctionInfo(),
+		GetParamsCmd(),
+	)
+	return cmd
+}
+
+// GetAuctionInfo queries the current auction information
+func GetAuctionInfo() *cobra.Command {
```

Comment:
consistency with other commands:
```suggestion
// GetAuctionInfoCmd queries the current auction information
func GetAuctionInfoCmd() *cobra.Command {
```